src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -164,6 +166,11 @@ function IOURequestStepWaypoint({
             onEntryTransitionEnd={() => textInput.current?.focus()}
             shouldEnableMaxHeight
             testID={IOURequestStepWaypoint.displayName}
+            style={[styles.overflowHidden]}
+            // This keeps the submit button at the bottom of the screen on iOS
+            shouldEnableKeyboardAvoidingView={Platform.OS !== 'ios'}
```

Comment:
We try to avoid usage of `Platform.OS` directly inside the components, take a look at 
https://github.com/expensify/app?tab=readme-ov-file#platform-specific-file-extensions

> In most cases, the code written for this repo should be platform-independent. In such cases, each module should have a single file, index.js, which defines the module's exports. There are, however, some cases in which a feature is intrinsically tied to the underlying platform. In such cases, the following file extensions can be used to export platform-specific code from a module:
>- Mobile => index.native.js
>- iOS Native App/Android Native App => index.ios.js/index.android.js
>- Web => index.website.js
>- Desktop => index.desktop.js